ah_bootstrap.py

```diff
@@ -38,63 +38,42 @@
 
 import contextlib
 import errno
-import imp
 import io
 import locale
 import os
 import re
 import subprocess as sp
 import sys
 
+__minimum_python_version__ = (3, 5)
```

Comment:
Should this be `(3, 6)`? (is this file just an update from an external source?)

Reply:
yeah this is just an update from astropy_helpers, we will add our own version of this when I get around to it lol.